pkg/controller/clusterapi.go

```diff
@@ -91,3 +92,76 @@ func GetMachineDeployment(ctx context.Context, client client.Client, machineDepl
 	}
 	return machineDeployment, nil
 }
+
+// GetMachines reads a list cluster-api Machines for an eks-a cluster using a kube client.
+func GetMachines(ctx context.Context, c client.Client, cluster *anywherev1.Cluster) ([]clusterv1.Machine, error) {
+	// "get", capiMachinesType, "-o", "json", "--kubeconfig", cluster.KubeconfigFile,
+	// "--selector=cluster.x-k8s.io/cluster-name=" + clusterName,
+	// "--namespace", constants.EksaSystemNamespace,
+	machines := &clusterv1.MachineList{}
+	err := c.List(ctx, machines, client.MatchingLabels{clusterv1.ClusterNameLabel: cluster.Name}, client.InNamespace(constants.EksaSystemNamespace))
```

Comment:
Can we use the unauth client to list these machines?

Ref: https://github.com/aws/eks-anywhere/blob/main/pkg/clients/kubernetes/unauth.go#L66